scss-partials/_general.scss

```diff
@@ -0,0 +1,221 @@
+        height: 56px;
+    }
+
+    @include media ( ">=lg" ) {
+        height: 125px;
+    }
+
+    &__container {
+        max-width: 1264px;
+        margin: 0 auto;
+        width: 100%;
+    }
+}
+
+.hamburger {
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0;
+
+    @include media ( ">=lg" ) {
+        display: none;
+    }
+
+    &__img {
+        margin: 16px 16px 0px;
+        cursor: pointer;
+    
+        @include media ( ">=md" ) {
+            margin: 19px 16px 0px;
+        }
+    }
+}
+
+.nav {
+    line-height: 20px;
+    color: $tws_black;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 10px;
+    width: 100%;
+    position: relative;
+    height: 48px;
+    font-family: $font-regular;
+
+    @include media ( ">=md" ) {
+        height: 56px;
+    }
+
+    @include media ( ">=lg" ) {
+        line-height: 24px;
+        height: 61px;
+    }
+
+    &__column {
+        width: 33.33%;
+    }    
+}
+
+.logo {
+    text-align: center;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    &__bow {
+        height: 10px;
+        width: 19px;
+        margin: 12px 0px 0px;
+    
+        @include media ( ">=lg" ) {
+            height: 19px;
+            width: 37px;
+            margin: 16px 0px 0px;
+        }
+    }
+    
+    &__name {
+        width: 117px;
+        height: 6.5px;
+        margin: 7px 0px 0px;
+    
+        @include media ( ">=lg" ) {
+            width: 220px;
+            height: 12px;
+            margin: 13px 0px 0px;
+        }
+    }
+}
+
+.control {
+    text-align-last: right;
+
+    &__option {
+        text-align: center;
+        position: relative;
+        display: inline-block;
+        cursor: pointer;
+        height: 20px;
+        text-align: center;
+        margin: 14px 56px 0px 0px;
+    
+        @include media ( ">=md" ) {
+            margin: 16px 56px 0px 0px;
+        }
+    
+        @include media ( ">=lg" ) {
+            margin: 24px 16px 0px 0px;
+            height: 24px;
+        }
+    }
+}
+
+.link {
+    cursor: pointer;
+
+    &:hover { text-decoration: underline; }
+}
+
+.create {
+    background: $tws_blue;
+    color: $tws_white;
+    text-transform: uppercase;
+    width: 94px;
+    padding: 4px 16px;
+    margin-right: 32px;
+
+    &:hover { background: $tws_coral; }
+
+    &__list {
+        text-align: center;
+        font-weight: bold;
+        letter-spacing: 0.4px;
+    }
+}
+
+.create, .findCouple {
+    display: none;
+
+    @include media ( ">=lg" ) {
+        display: inline-block;
+    }
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0;
+    font-family: $font-regular;
+
+    @include media ( "<lg" ) {
+        display: none;
+    }
+
+    &__list {
+        display: flex;
+        justify-content: space-evenly;
+        width: 100%;
+        padding: 0px;
+        margin: 0px;
+    
+        &--item {
```

Comment:
I know this is an old piece of code but have just noticed it - the list item should rather have a class `&__list-item`. &-- bit indicates it's a modifier and I don't think it is. For instance if you had a list that appears on a page in two places - the only difference is that one of them has a red font and one has black, you could have something like:
```
.list {
...some styles;

   &--primary {
     color: black;
     }
     
    &--secondary {
      color: red
     } 
 }
```
So the double dash means a modifier class, you could read about it in BEM documentation. If you have a classname that uses two or more words but is not a modified, please use single dash (.list-item) or even camelCase (.listItem)

Reply:
Yes, understood. In this case i've used menu__list for the ul classname and this has different properties to the list items, how would I name those? menu__list to be menuList instead and then menuList__item for the list items?

Reply:
it can be menu__list and menu__list-item or menuList (if list is not an element of the menu block style-wise) and then menuList__item. Up to you and depends on what you consider a block and what an element. Just wanted to point out that double dash is for modifiers - you can keep all your current classes and only change .menu__list--item to be .menu__list-item or .menu__listItem. That's all :)

Reply:
Ok great, so can I use 'menu' as the block, 'menu__list' and 'menu__list-item' as elements? Can I change the items to have both 'menu__list' and 'menu__list-item' elements on the same line as these have differing attributes?

Reply:
Actually, am I ok to just have 'menu__item' instead of 'menu__list-item'? The UL is still 'menu__list' and then the LI would be 'menu__item'?

Reply:
yes, that's absolutely fine :)